example/capsnet/README.md

```diff
@@ -0,0 +1,32 @@
+**CapsNet-MXNet**
+=========================================
+
+This example is MXNet implementation of [CapsNet](https://arxiv.org/abs/1710.09829):  
+Sara Sabour, Nicholas Frosst, Geoffrey E Hinton. Dynamic Routing Between Capsules. NIPS 2017
+- The Current best test error is 0.5%  
+
+Due to the permission issue, this example is maintained in this [repository](https://github.com/samsungsds-rnd/capsnet.mxnet) separately.

```

Comment:
what does this mean?

Reply:
If you mean the last line, Pull Request can takes days to accept, meanwhile some follow-up changes can be updated to our temporary repository https://github.com/samsungsds-rnd/capsnet.mxnet.